run.py

```diff
@@ -0,0 +1,29 @@
+from lu_vp_detect import VPDetection
```

Comment:
Not really liking this test file.  It only runs JPEG files.  Just have it work on the example image that I have in my repo.